contracts/examples/multisig/mandos/deployOtherMultisig.scen.json

```
@@ -0,0 +1,144 @@
+{
+    "gasSchedule": "v4",
+    "steps": [
+        {
+            "step": "externalSteps",
+            "path": "steps/init_accounts.steps.json"
+        },
+        {
+            "step": "externalSteps",
+            "path": "steps/deploy.steps.json"
+        },
+        {
+            "step": "externalSteps",
+            "path": "steps/add_prop.steps.json"
+        },
+        {
+            "step": "externalSteps",
+            "path": "steps/add_bm.steps.json"
+        },
+        {
+            "step": "scCall",
+            "txId": "proposeSCDeployFromSource",
+            "tx": {
+                "from": "address:paul",
+                "to": "sc:multisig",
+                "function": "proposeSCDeployFromSource",
+                "arguments": [
+                    "0",
+                    "sc:multisig",
+                    "false",
+                    "false",
+                    "false",
+                    "1",
+                    "address:paul"
+                ],
```

The contract having its deploy proposed here is also used when upgrading in the `upgrade.scen.json` test.
As such, it should be upgradable:
```suggestion
                "function": "proposeSCDeployFromSource",
                "arguments": [
                    "0",
                    "sc:multisig",
                    "true",
                    "false",
                    "false",
                    "1",
                    "address:paul"
                ],
```